core/controllers/topic_editor_test.py

```diff
@@ -418,6 +575,55 @@ def test_editable_topic_handler_put(self):
                     feconf.TOPIC_EDITOR_DATA_URL_PREFIX, self.topic_id),
                 change_cmd, csrf_token=csrf_token, expected_status_int=401)
 
+            # Check that topic can not be edited when version is None.
+            self.login(self.ADMIN_EMAIL)
+
+            json_response = self.put_json(
+                '%s/%s' % (
+                    feconf.TOPIC_EDITOR_DATA_URL_PREFIX, self.topic_id),
+                {'version': None}, csrf_token=csrf_token,
+                expected_status_int=400)
+            self.assertEqual(
+                json_response['error'],
+                'Invalid POST request: a version must be specified.')
+
+            self.logout()
+
+            # Check topic can not be edited when payload version differs from
+            # topic version.
+            self.login(self.ADMIN_EMAIL)
+
+            topic_id_1 = topic_services.get_new_topic_id()
+            self.save_new_topic(
+                topic_id_1, self.admin_id, 'Name 1', 'Description 1', [], [],
```

Comment:
Please use kwargs on the list to help readers understand what they are.

Reply:
But the arguments in save_new_topic are not keyword arguments and we should avoid the use of args-name for a non-keyword argument. You can see this issue -> https://github.com/oppia/oppia/issues/6067

Reply:
Added my thoughts to the issue, TL;DR: I strongly disagree we be avoiding using kwarg names.